src/borg/archiver.py

```diff
@@ -2254,6 +2254,15 @@ def process_epilog(epilog):
         '\*/.bundler/gems' to get the same effect. See ``borg help patterns`` for
         more information.
 
+        In addition to using ``--exclude`` patterns, it is possible to use
+        ``--exclude-if-present`` to specify the name of a filesystem object (e.g. a file
+        or folder name) which, when contained within another folder, will prevent the
+        containing folder from being backed up.  By default, the containing folder and
+        all of its contents will be omitted from the backup.  If, however, you wish to
+        only include the objects specified by ``--exclude-if-present`` in your backup,
+        and not include any other contents of the containing folder, this can be enabled
+        through using the ``--keep-exclude-tags`` option.
```

Comment:
Not sure about the examples section, because besides this text (and the short description) there is no documentation at all about these options.  I see this as explaining in detail how the options work, not as examples.

And yes, motivation is not included, and while I agree that it could be nice, I'm not motivated ;) to add it.